### PR TITLE
Contribution quality principles

### PR DESCRIPTION
**Pull request by raquelbreternitz**
Adding in a md table showcasing the new Contribution Quality principles:

Screen Shot 2022-08-03 at 4 20 54 PM

### CODE REVIEW

File: polaris.shopify.com/content/contributing/index.md [hunk 1]
```diff
@@ -17,11 +17,23 @@ Contribution to Polaris looks different depending on whether you work at Shopify
...
+| **🏗️ Consider the impact on the system** <br>Contributions to Polaris should apply to multiple admin surfaces. They should not be unique solutions with only one use case. Read [Designing with a system](/contributing/designing-with-a-system), and reach out to the Polaris team for system advice or guidance. | **🔬 Be clear and detailed** <br>Submit a short summary of your contribution, along with relevant links, artifacts, and context. State your expectations, understanding that we must prioritize certain business and system needs. If you can ship in collaboration with another product team, tag them and let the Polaris team know. Capture contribution details with [GitHub issue templates](https://github.com/Shopify/polaris/issues/new/choose). |
+|                                                                                                                                                                                                                                                                                                                    |
```
Comment (alex-page) [inline polaris.shopify.com/content/contributing/index.md]: Love this. What about just using a level three header for `### Preparing contributions` and put the content below it. Tables are not the best unless they have tabular data.

**Review by alex-page — APPROVED** [review]
Approving because this is awesome and I would be happy for it to ship as is.

**Review by jjgali — APPROVED** [review]

File: polaris.shopify.com/content/contributing/index.md [hunk 2]
```diff
@@ -15,13 +15,25 @@ Contribution to Polaris looks different depending on whether you work at Shopify
...
```
Comment (chloerice) [inline polaris.shopify.com/content/contributing/index.md]: (Removes a couple of extra pipe characters that are adding an empty table row to the bottom)

```suggestion
```

**Review by chloerice — APPROVED** [review]
This looks great!! 👀 🎉 Just need to remove the two extra `|` pipe characters at the end of the table 🚢

File: polaris.shopify.com/content/contributing/index.md (hunk 2 above)
Comment (raquelbreternitz) [reply polaris.shopify.com/content/contributing/index.md]: ooh, thank you!

File: polaris.shopify.com/content/contributing/index.md (hunk 1 above)
Comment (raquelbreternitz) [reply polaris.shopify.com/content/contributing/index.md]: @alex-page Yeah, it's not ideal but we felt without the table giving us the artificial 2-column layout it bloats the page to be far too long. This at least helps it be more scannable.

Comment (alex-page) [reply polaris.shopify.com/content/contributing/index.md]: Awesome. Thanks for the context 🚀

**Review by selenehinkley — APPROVED** [review]

**Review by tracytees — APPROVED** [review]

**Pull request merged**